packages/api/src/feature_flag/validators.ts

```diff
@@ -0,0 +1,19 @@
+import Joi from "joi";
+import type { CreateFeatureFlagRequest } from "./models";
+import { fieldsFromAdminAuthentication } from "../validators";
+
+export function validateCreateFeatureFlagRequest(
+  data: unknown
+): asserts data is CreateFeatureFlagRequest {
+  const validation = Joi.object()
+    .keys({
+      ...fieldsFromAdminAuthentication,
+      name: Joi.string().required().max(255),
+      description: Joi.string().max(10000),
```

Comment:
No need for the maximum lengths here; we're storing these at PostgreSQL TEXT, which does not have a character limit (technically it can't hold strings larger than 1 GB, but I'm alright with people getting a 500 error if they try to submit a > 1 GB string here)